geometry2/0af76d5/0af76d5.bug

```diff
@@ -74,4 +74,4 @@ fault-codes:
   - CONCURRENCY:NO-SYNC
 failure-codes:
   - SOFTWARE:BEHAVIORAL
-  - SYSTEM:LOSS-OF-CONTROL
+  - SYSTEM:LOSS-OF-FUNCTIONALITY
```

Comment:
@gavanderhoorn we're unsure whether this leads to either: (a) a crash at run-time, or (b) using a meaningless/corrupted transform. If the former, then we should go with LOSS-OF-FUNCTIONALITY. If the latter, we should go with BEHAVING-INCORRECTLY.

Reply:
@ChrisTimperley @gavanderhoorn do we have a resolution on this?

Reply:
If @gavanderhoorn doesn't get back on this soon, we can create a separate issue to keep track on this question so that the rest of the PR can go ahead.

Reply:
@ChrisTimperley apart from this bug, others have been updated. Should be ready to be merged if it's OK.

Reply:
https://github.com/robust-rosin/robust/issues/447

Reply:
It looks like we both created an issue 😅 https://github.com/robust-rosin/robust/issues/446

Reply:
:D I will close mine